lexer/lex.go

```diff
@@ -122,10 +134,19 @@ func (l *lexer) initialState() (Tokval, lexerState) {
 	return l.identifierState()
 }
 
+func (l *lexer) lineTerminatorToken() Tokval {
```

Comment:
You changed LineTerminator token to Newline, but this function really should remains the same? `newlineToken()`.